variables.tf

```diff
@@ -41,4 +41,10 @@ variable "tags" {
   description = "Tags to apply to the resources created by this module"
   type        = map(string)
   default     = {}
+}
+
+variable "defer_role_creation" {
```

Comment:
This new variable missing in readme and in template

Reply:
Yes. We want to keep it simple for now. Just incase, if someone wants to re-apply and not use lambda, we can use this flag.